docs/FAQ.md

```diff
@@ -515,6 +515,27 @@ by `jj rebase` to rebase the changes in a commit. It's used in `jj log` to
 indicate which commits are empty. It's used in the `files()` revset function
 (and by `jj log <path>`) to find commits that modify a certain path. And so on.
 
+## How do I revert a merge commit? `jj revert <merge>` does nothing
+
+Instead use `jj restore --from <first parent>` to revert the changes merged
+from the second parent.
+
+Example:
+```text
+@
+|
+C
+| \
+B D
+|/
+A
+```
+To revert the merge in `C`, `jj restore --from B` and then describe the message
+with something like `jj desc -m "Revert merge <C's commit-id>`. As this removes
+the parents edge to `D`, so the commits tree is reset to a state without its
+changes.
```

Comment:
Same here. "removes the parents edge to `D`" wasn't easy to understand.

Reply:
A possible replacement:

> Now, commit `@` undoes the merge of `D` into `B`. If necessary, you can now rebase it elsewhere, e.g. `jj rebase -r @ -d main`.

Reply:
used Ilya's suggestion